style.css

```diff
@@ -436,6 +454,188 @@ i {
   float: left;
 }
 
+/* Responsive design */
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Your app is nicely responsive I must confess, but there few things you need to pay attention to according to the given template. I noticed that @exactly `min-width: 768px`, the `let's connect` link is overlapping with other text **(index.html line 48)**. Kindly fix this for great user experience. 👍